goentri.com.showit.json

```diff
@@ -0,0 +1,86 @@
+{
+  "providerId": "goentri.com",
+  "providerName": "Entri",
+  "serviceId": "showit",
+  "serviceName": "Showit",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows user to easily set up dkim record using Entri",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
+  "variableDescription": "cnamePointsTo is the URL to where the www CNAME is pointed, ipAdress is the IP required for the A record,  and host is the host name ",
+  "sharedProviderName": true,
+  "hostRequired": false,
+  "records": [
+    {
+      "type": "A",
+      "host": "@",
+      "groupId": "a1",
+      "pointsTo": "%ipAdress%",
+      "ttl": 3600
+    },
+    {
+      "type": "A",
+      "host": "%host%",
```

Comment:
`%host%` is a reserved name, so using it will create a subdomain %host%.%host%. I guess this is not the intention.